cmd/dfget/app/root.go

```diff
@@ -234,8 +242,8 @@ func initFlags() {
 			"\nin this way, different but actually the same URLs can reuse the same downloading task")
 	flagSet.StringSliceVar(&cfg.Header, "header", nil,
 		"http header, eg: --header='Accept: *' --header='Host: abc'")
-	flagSet.StringSliceVarP(&cfg.Nodes, "node", "n", nil,
-		"specify the addresses(host:port) of supernodes")
+	flagSet.StringSliceVarP(&cfg.Supernodes, "node", "n", nil,
```

Comment:
Nice! For this specific flag type, isn't it better to introduce a new [flag.Value][1] for it?

[1]: https://golang.org/pkg/flag/#Value